massa-node/src/operation_injector.rs

```diff
@@ -113,7 +114,7 @@ pub fn start_operation_injector(
             protocol_controller
                 .propagate_operations(storage.clone())
                 .unwrap();
-            std::thread::sleep(Duration::from_secs(1));
+            std::thread::sleep(Duration::from_secs(1).saturating_sub(now.elapsed()));
```

Comment:
This is a fix for the operation injector, I'll take it out of this PR and create a new one especially for this

Reply:
You can let it there it's ok

Reply:
PR: #4194